route/build/src/read_control.f90

```diff
@@ -129,6 +132,7 @@ SUBROUTINE read_control(ctl_fname, err, message)
    case('<dt_qsim>');              read(cData,*,iostat=io_error) dt                    ! time interval of the simulation [sec] (To-do: change dt to dt_sim)
    case('<floodplain>');           read(cData,*,iostat=io_error) floodplain            ! logical: floodwater is computed, otherwise, channel is unlimited bank depth
    case('<hw_drain_point>');       read(cData,*,iostat=io_error) hw_drain_point        ! integer: how to add inst. runoff in reach for headwater HRUs. 1->top of reach, 2->bottom of reach (default)
+   case('<tracer>');               read(cData,*,iostat=io_error) tracer                ! logical; tracer is activated to compute a constituent transport
```

Comment:
Mention solute in the comment.

Reply:
comment revised to `tracer is activated to compute a solute or constituent transport. Note: currently used for DOC`